csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst/MakeConstAnalyzer.cs

```diff
@@ -28,10 +28,11 @@ public class MakeConstAnalyzer : DiagnosticAnalyzer
 
         public override void Initialize(AnalysisContext context)
         {
-            // See https://github.com/dotnet/roslyn/blob/master/docs/analyzers/Analyzer%20Actions%20Semantics.md for more information
-            // <SnippetRegisterNodeAction>
```

Comment:
I'm confused by this set of changes, unless I'm missing something. You've moved the opening snippet tag from the old line 32 to the new line 35, and you've removed the closing snippet tag (which was on the old line 34). This will gneerate a build warning, and code won't appear in the build.

Reply:
I didn't realize how the Snippet comments works. My intent was to add to lines to the snippet. I will commit it fix later today.